biocrnpyler/dna_assembly_promoter.py

```diff
@@ -164,4 +164,173 @@ def update_reactions(self, **keywords):
         reactions = [] #a list of reactions must be returned
         reactions += mech_tx.update_reactions(dna = self.assembly.dna, transcript = self.transcript, repressor = self.repressor, 
                                              component = self, part_id = self.name+"_"+self.repressor.name, **keywords)
+        return reactions
+
+class CombinatorialPromoter(Promoter):
+    def __init__(self, name, regulators, leak = True, assembly = None,
+                 transcript = None, length = 0, mechanisms = {},
+                 parameters = {},tx_capable_list = None,cooperativity = None, **keywords):
```

Comment:
Hi Andrey, 
you might want to change the argument for the parameter dictionary like this:
parameters = None
to avoid the problem in #76